CHANGELOG.adoc

```diff
@@ -11,6 +11,7 @@
 - Switch to Sirius Web 2024.5.0
 - Switch to @ObeoNetwork/gantt-task-react 0.4.9
 - Add dependency to org.apache.commons.commons-text 1.10.0
+- Add dependency to Cypress 12.11.0
```

Comment:
You will also have to move this in the new v2024.7.0 section in the changelog after the merge of https://github.com/eclipse-syson/syson/pull/318, which should occurs tomorrow morning first task

Reply:
done